src/docs/asciidoc/index.adoc

```diff
@@ -1,8 +1,8 @@
 = Micronaut Gradle plugin
-:native-build-tools-version: 0.9.11
+:native-build-tools-version: 0.9.13
 :kotlin-version: 1.6.21
 :micronaut-version: 3.5.1
-:gradle-version: 7.4.1
+:gradle-version: 7.5.1
```

Comment:
I have created a separate PR for this: https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/535